cmd/tfsec/main.go

```diff
@@ -47,14 +51,16 @@ var ignoreWarnings = false
 var ignoreInfo = false
 var allDirs = false
 var runStatistics bool
+var stopOnHCLError bool
```

Comment:
I think this ought to be default to true and changed to `ignore-hcl-errors`? You'd need to know you had an hcl to stop on them

Reply:
I think so too, was worried about breaking BC though - people's builds will now start failing if they have invalid HCL in there. But maybe that is a good thing... :sweat_smile:

Reply:
there are a few scenarios -

1. user has broken terraform which has hcl error and they don't know (not sure how because it probably wouldn't deploy)
2. tfsec stops on error incorrectly, we get an issue and look at it on merit